core/src/main/java/ai/timefold/solver/core/impl/domain/variable/declarative/LoopedTracker.java

```diff
@@ -34,23 +35,24 @@ public void mark(int node, LoopedStatus status) {
         looped.set(node, status.ordinal());
     }
 
-    public boolean isEntityLooped(BaseTopologicalOrderGraph graph, int entityId, boolean wasEntityLooped) {
+    public boolean isEntityInconsistent(BaseTopologicalOrderGraph graph, int entityId,
+            @Nullable Boolean wasEntityInconsistent) {
         for (var entityNode : entityIdToNodes[entityId]) {
             if (graph.isLooped(this, entityNode)) {
```

Comment:
Question: do we want to keep `loop` in the graph, but use consistency everywhere else? This proliferates elsewhere in the code - `LoopedTracker`, `looped.set(...)`, ... 

I understand that, within the context of the graph, "loop" is more descriptive than "inconsistent". Still, somehow, I don't like mixing the two. Dunno.

Reply:
I prefer looped with graphs, since it an implementation detail, and since "inconsistent" is vague when talking about a graph.